### ISSUE TITLE
Option to allow extreme values to be represented by an asterisk (*) in box plots

### ISSUE THREAD

**Issue by amergin** [dc-js/dc.js#1162]
Feature request: add option to box plots that allows extreme values to be represented by an asterisk (*). An extreme value is defined that is situated more than three interquartile ranges from the edge of the box.

**Comment by gordonwoodhull** [comment]
That's a great idea @amergin - any chance you'd like to contribute it? 

I don't think this can be done as a renderlet because it changes the minimum/maximum domain (so the stars would often not be shown if elastic is used).

**Comment by amergin** [comment]
I'm not promising anything. I could look into it at some point if I need it in some project. Just thinking ahead - since the asterisk notation is widely used in statistical programs, someone will eventually need it.

**Comment by gordonwoodhull** [comment]
Yes, I agree it's a standard feature of these visualizations and we should support it.

**Comment by gordonwoodhull** [comment]
If anyone's interested in this, probably the most general thing would be to fork [box.js](https://github.com/d3/d3-plugins/blob/master/box/box.js) and use d3 symbols instead of [hardcoding the circle shape](https://github.com/d3/d3-plugins/blob/40f8b3b91e67719f58408732d7ddae94cafa559a/box/box.js#L158).

**Comment by cwolcott** [comment]
Similar to this, I wanted to allow the developer to define the symbol to use for mild outliers (1.5 * IQR) and I guess now extreme outliers (3.0 * IQR).  I have not had the pleasure of writing code to use symbols yet, but I am sure I can find examples in other charts.

Here is the current work I am doing on boxPlots:  #1120
